internal/dependencytrack/policy.go

```diff
@@ -56,12 +56,39 @@ type ViolationAnalysis struct {
 	IsSuppressed  bool   `json:"isSuppressed,omitempty"`
 }
 
+// GetAllViolations returns violations for the entire portfolio. Suppressed
+// violations are omitted unless suppressed is true
+func (c *Client) GetAllViolations(suppressed bool) ([]*PolicyViolation, error) {
+	// dependency track per default only returns a 100 items per get, therefore we need to iterate over all PolicyViolation pages to get all PolicyViolation
+
+	// all PolicyViolation found in pagination
+	allPolicyViolations := []*PolicyViolation{}
+	// the last project found in the last pagination page result
+	lastPaginationPage := 1
+	// state var to show if all PolicyViolation projects where found
+	foundAll := false
+
+	for !foundAll {
+		req, err := c.GetViolations(suppressed, lastPaginationPage, 100)
+		if err != nil {
+			return nil, err
+		}
+		if len(req) == 0 {
```

Comment:
The number of total items will not necessarily be a multiple of 100, which means the 'last page' will have less than 100 results. 

If we change this check to `len(req) < 100` (and move some of the logic around) I think we can avoid an extra request to the API in most cases.